website/app/components/doc/image-caption/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div class={{this.classNames}} ...attributes>
+  {{yield}}
```

Comment:
@andgen404 do we allow to have also structured content inside a caption (eg. a bold text, a link, a list, etc) or should it be text-only?
- if it's text only, instead of yielding we should expose a `@text` argument to prevent passing structured content
- if we allow to pass structured content, then we should provide basic formatting (expecially for the link)

Reply:
@curthashicorp on a second thought: I did this test:
```
<Doc::ImageCaption>
Lorem ipsum
- ipsum
- dolor
</Doc::ImageCaption>
```
and guess what?
<img width="802" alt="screenshot_4016" src="https://github.com/user-attachments/assets/1ccf9174-bf99-4e2f-a9b1-38604541f885">

So to avoid people adding markdown content that works in some cases but not others, better to be on the safe side and use just a `@text `argument, instead of the `yield`.

Reply:
Great point.  This would add a lot of confusion for content authoring.  I have implement @text argument.